modules/timepoint_list/templates/menu_timepoint_list.tpl

```diff
@@ -39,7 +39,7 @@
     <tbody>
     {section name=timepoint loop=$timePoints}
         <tr>
-            <td><a href="{$baseurl}/{$candID}/{$timePoints[timepoint].SessionID}/">{$timePoints[timepoint].Visit_label}</a></td>
+            <td><a href="{$baseurl}/instrument_list/?candID={$candID}&sessionID={$timePoints[timepoint].SessionID}/">{$timePoints[timepoint].Visit_label}</a></td>
```

Comment:
I think the original URL should work. There should already be a rewrite rule which writes a CandID to the timepoint_list, and the CandID/Visit_label to the instrument_list.